feat(minigo): add stdlib test for path/filepath (#414)
podhmo/go-scan: TODO.md

### Future Interpreter Enhancements (for Stdlib Support)
- [x] **Implement two-pass evaluation for top-level declarations**: To fix the "Sequential Declaration Processing" limitation, modify the interpreter to first scan all top-level declarations (types, funcs, vars, consts) in a package before evaluating any code.

> - [ ] **Add support for string indexing**: Enhance the evaluator to handle the index operator (`s[i]`) on string objects. This is required for direct source interpretation of the `strings` package.

Reviewer: ```suggestion
- [x] **Add support for string indexing**: Enhance the evaluator to handle the index operator (`s[i]`) on string objects. 
```
